aboutus.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+	<head> <!-- Page's intelligence -->
+		<title>SILK : natural cosmetic products</title>
+		<meta name="description" content="This is SILK, naturals and efficient products">
+		<meta charset="utf-8">
+		<link href="https://fonts.googleapis.com/css2?family=Alike&family=Antic+Didone&family=Comic+Neue&display=swap" rel="stylesheet">
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css">
+		<link rel="stylesheet" href="stylecream.css">
+		<link rel="stylesheet" href="aboutus.css"
+	</head>
+
+		<body>
+			<div class="container">
+				<div class="text-center"> <!-- Introduction -->
+					<h1>SILK</h1> <h2> Paris</h2> 

```

Comment:
I don't like that you uses all caps for this. Maybe we should use sentence case but make it bold.